diff --git forkSrcPrefix/Makefile forkDstPrefix/Makefile
index 002669bcd172c7b087000c7d258b325511909680..a7e4041afa79889ee568ccd5d51d6d3ef87191b9 100644
--- forkSrcPrefix/Makefile
+++ forkDstPrefix/Makefile
@@ -1,11 +1,11 @@
-OS = windows
+OS = linux
 BUILD_TYPE = release_static
 
 OUTPUT := ps3netsrv
 OBJS = src/main.o src/padlock.o src/aes.o src/compat.o src/mem.o src/File.o src/VIsoFile2.o
 
-CFLAGS = -Wall -Wno-format -I./include -std=gnu99 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DPOLARSSL
-CPPFLAGS += -Wall -Wno-format -I./include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DPOLARSSL
++CFLAGS = -Wall -I./include -std=gnu99 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DPOLARSSL
++CPPFLAGS += -Wall -I./include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DPOLARSSL
 
 #CFLAGS += -Doff64_t=off_t
 #CPPFLAGS += -Doff64_t=off_t
